src/transaction/log_manager.c

```diff
@@ -7740,11 +7736,9 @@ log_sysop_do_postpone (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_REC_SYSOP_E
   sysop_start_postpone.posp_lsa = *LOG_TDES_LAST_SYSOP_POSP_LSA (tdes);
   log_append_sysop_start_postpone (thread_p, tdes, &sysop_start_postpone, data_size, data);
 
-  if (VACUUM_IS_THREAD_VACUUM (thread_p)
-      && vacuum_do_postpone_from_cache (thread_p, LOG_TDES_LAST_SYSOP_POSP_LSA (tdes)))
+  if (tdes->m_log_postpone_cache.do_postpone (*thread_p, LOG_TDES_LAST_SYSOP (tdes)->posp_lsa))
```

Comment:
Hope to unify `LOG_TDES_LAST_SYSOP (tdes)->posp_lsa` and `LOG_TDES_LAST_SYSOP_POSP_LSA (tdes)` from:
```c
  if (tdes->m_log_postpone_cache.do_postpone (*thread_p, LOG_TDES_LAST_SYSOP (tdes)->posp_lsa))
    {
      /* Do postpone was run from cached postpone entries. */
      tdes->state = save_state;
      return;
    }

  log_do_postpone (thread_p, tdes, LOG_TDES_LAST_SYSOP_POSP_LSA (tdes));
```

Reply:
fixed